packages/sit-onyx/src/components/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -139,6 +139,7 @@ const shouldShowExternalIcon = (args: OnyxNavItemProps) => {
 
     &__icon {
       align-self: flex-start;
+      margin-left: -0.5rem;
```

Comment:
I'd prefer to create a scss Variable and use it for the regular gap and in here so that they are related when me might change the gap size in the future :) Something like this:

```scss
$gap: var(--onyx-scaing-sm);

&_trigger {
  gap: $gap;
}

&__icon {
  margin-left: calc(-1 * $gap);
}
```